src/components/renderers/layouts/masterdetail/masterdetail-directives.ts

```diff
@@ -31,13 +58,19 @@ class MasterDetailController extends AbstractControl {
 class MasterDetailCollectionController {
     static $inject = ['$scope'];
     public instance: any;
-    public properties: any;
+    public schema: any;
+    public labelprovider: LabelProvider;
```

Comment:
I'd prefer for camel case/kebap case (for HTML) naming: `labelProvider`, `imageProvider`, ...